doc/03-omnifest/01-directive.md

```diff
@@ -196,26 +196,66 @@ otk.meta.kiwi:
 
 ## `otk.customization`
 
+A `otk.customization` is distinct from a `otk.define` because:
+- End-users will supply them
```

Comment:
We want to be able to inspect omnifests to see which customizations are allowed; we want to provide default values for customizations (when not enabled) which we don't support in defines.

Reply:
Both could be done without a new concept (not saying we should, my gut feeling is that we want customizations but I think it's important to figure out why exactly), we could have in our default
```yaml
otk.define:
 customizations.hostname: "localhost.localdomain"
```
and then the user provides "config.yaml" with:
```
otk.define:
  customizations.hostname: "my.hostname"
```

We could know what customizations are allowed via convention (anyhting defined via otk.define with "customization.*" is a customization).

Again, not saying we should do this but I'm still struggling to have a clear picture in my head.